docs/source/overview/tutorials/fonll.ipynb

```diff
@@ -0,0 +1,280 @@
+    "        'interpolation_polynomial_degree': 4, \n",
+    "        # Interpolation: if True use log interpolation\n",
+    "        'interpolation_is_log': True, \n",
+    "        # Scale variations: \"None\", \"expanded\", \"exponentiated\"\n",
+    "        'scvar_method': None, \n",
+    "        # Backward evolution matching inversion: \"None\", \"expanded\", \"exact\"\n",
+    "        'inversion_method': None, \n",
+    "        # Number of parallel cores to use: 0 = all, 1 = one, -1 = all but one ...\n",
+    "        'n_integration_cores': 0, \n",
+    "        # Polarized evolution ? \n",
+    "        'polarized': False, \n",
+    "        # Time like evolution ? \n",
+    "        'time_like': False\n",
+    "    }, \n",
+    "    # Debug options\n",
+    "    'debug': {\n",
+    "        'skip_singlet': False, \n",
+    "        'skip_non_singlet': False\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2907d131",
+   "metadata": {},
+   "source": [
+    "Then we compute the necessary coexisting flavor number PDFs starting from a given PDF defined for some value of $Q$ and corresponding $n_f$\n",
+    "using `EKO` (for more detailed information on the use of `EKO` we refer to the [documentation](https://eko.readthedocs.io) ).\n",
+    "Here, we show how to evolve `NNPDF40_nnlo_as_01180` to produce three PDFs in FFNS3, FFNS4, and FFNS5 respectively."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b001ecf8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import lhapdf\n",
+    "from eko.io import runcards\n",
+    "from ekobox.evol_pdf import evolve_pdfs\n",
+    "\n",
+    "# Define the PDF for which we want to make the prediction\n",
+    "input_pdf = lhapdf.mkPDF(\"NNPDF40_nnlo_as_01180\", 0)\n",
+    "\n",
+    "# Build eko-like runcards objects\n",
+    "theory_card = runcards.Legacy(theorycard, {}).new_theory\n",
+    "operators_card = runcards.OperatorCard.from_dict(operatorcard)\n",
+    "\n",
+    "# Iterate the flavor numbers\n",
+    "for nf in [3, 4, 5]:\n",
+    "    # Update the nf value in the mugrid\n",
+    "    operators_card.mugrid = [(mu, nf) for (mu,_) in operators_card.mugrid]\n",
+    "\n",
+    "    # Evolve the LHAPDF grid and store the output\n",
+    "    evolve_pdfs(\n",
+    "        initial_PDF_list=[input_pdf],\n",
+    "        theory_card=theory_card,\n",
+    "        operators_card=operators_card,\n",
+    "        store_path=f\"eko_nf{nf}.tar\",  # where to store the eko\n",
+    "        install=True, # Install in the evolved PDF in the LHAPDF directory\n",
+    "        name=f\"NNPDF40_evolved_nf{nf}\",  # name of the evolved pdf\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ec4dbbc",
+   "metadata": {},
+   "source": [
+    "Next, we compute the coefficient functions using `yadism` and convolve them with the three PDFs\n",
```

Comment:
At the beginning you call them "partonic x-sections". I would use one of the other name consistently.